neo/Plugins/Plugin.cs

```diff
@@ -12,6 +12,7 @@ public abstract class Plugin
     {
         public static readonly List<Plugin> Plugins = new List<Plugin>();
         private static readonly List<ILogPlugin> Loggers = new List<ILogPlugin>();
+        internal static readonly List<IStoragePlugin> Storages = new List<IStoragePlugin>();
```

Comment:
Follow on to earlier comment, why do we have a list of storage plugins?

Reply:
You may have multiple DLLs, but only one will be loaded.